insights/parsers/rsyslog_conf.py

```diff
@@ -10,70 +10,76 @@
 
 Due to high parsing complexity, this parser presents a simple line-based
 view of the file that meets the needs of the current rules.
-
-Example:
-    >>> content = '''
-    ... :fromhost-ip, regex, "10.0.0.[0-9]" /tmp/my_syslog.log
-    ... $ModLoad imtcp
-    ... $InputTCPServerRun 10514"
-    ... '''.strip()
-    >>> from insights.tests import context_wrap
-    >>> rsl = RsyslogConf(context_wrap(content))
-    >>> len(rsl)
-    3
-    >>> len(list(rsl))
-    3
-    >>> any('imtcp' in n for n in rsl)
-    True
 """
-from .. import Parser, parser, get_active_lines
-
-import re
+from .. import Parser, parser, get_active_lines, LegacyItemAccess
 from insights.specs import Specs
 
 
 @parser(Specs.rsyslog_conf)
-class RsyslogConf(Parser):
+class RsyslogConf(Parser, LegacyItemAccess):
```

Comment:
The `self.data` of this Parser is a list, `LegacyItemAccess` is not suitable to a list but a dict.

Would you mind change to mix it with a `list` (and remove the self.data)? 
```python
 class RsyslogConf(Parser, list):
```